asyncapi/quickbooks/Module.md

```diff
@@ -0,0 +1,64 @@
+## Overview
+The QuickBooks trigger allows you to listen to [QuickBooks webhook notifications](https://developer.intuit.com/app/developer/qbo/docs/develop/webhooks).
+
+After you subscribe for webhooks to receive event triggered callbacks for entities that your app needs to stay on top of. QuickBooks webhooks automatically notify you whenever data changes in your end-user’s QuickBooks Online company files.
```

Comment:
Shall we make this sentence more clear?

Reply:
Better to get this received by the doc team

Reply:
Better to get this module.md reviewed by the doc team

Reply:
Suggestion:

"After you subscribe for webhooks to receive event-triggered callbacks for entities that your application must listen to. QuickBooks webhooks automatically notify you whenever data changes in your end user’s QuickBooks Online company files."